packages/components/src/components/radio/radio.stories.ts

```diff
@@ -18,106 +24,74 @@ export default {
  */
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
-/**
- * Use the disabled attribute to disable a input radio. Clicks will be suppressed until the disabled state is removed
- */
-
-export const DisabledAndSize = {
-  name: 'Disabled × Size',
-  parameters: { controls: { exclude: ['disabled', 'size', 'default'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'disabled',
-          values: [false, true]
-        },
-        y: {
-          type: 'attribute',
-          name: 'size',
-          values: ['lg', 'sm']
-        }
-      },
-      constants: { type: 'attribute', name: 'disabled', value: true },
-      args
-    });
-  }
-};
-
 /**
  * Use the `size` attribute to change the size of the input radio. This attribute affects the font-size within the element, while the element itself remains the same size.
  */
 
 export const Size = {
-  parameters: { controls: { exclude: ['size'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+  name: 'Size',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-radio-group size="lg" value="1">
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2">Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+      </sd-radio-group>
+      <sd-radio-group size="sm" value="1">
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2">Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+      </sd-radio-group>
+    </div>
+  `
 };
 
 /**
- * Use the `invalid` attribute to mark the element is not in a format or a value the application will accept.
+ * Use the `disabled` attribute to disable a input radio.
  */
 
-export const Invalid = {
-  parameters: { controls: { exclude: ['invalid'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: [
-          { type: 'attribute', name: 'size' },
-          { type: 'attribute', name: 'disabled' }
-        ]
-      },
-      constants: { type: 'attribute', name: 'invalid', value: true },
-      args
-    });
-  }
+export const Disabled = {
+  name: 'Disabled',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-radio-group label="Disabled Radio in Group" boldLabel value="1">
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2" disabled>Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+      </sd-radio-group>
+    </div>
+  `
 };
 
 /**
- * Use the `base`, `control--unchecked`, `control--checked`, `checked` and `label` part selectors to customize the radio.
+ * The `invalid` attribute marks the radio as invalid. For an invalid radio-group (since radios always come in groups) an error-text underneath the group is mandatory.
```

Comment:
```suggestion
 * The `invalid` attribute marks the radio as invalid. 
 *
 * For an invalid radio-group (since radios always come in groups) an error-text underneath the group is mandatory.
```